bin/rspec

```diff
@@ -1,4 +1,5 @@
 #!/usr/bin/env ruby
```

Comment:
Please make sure all tests are passing and everything is green. Kindly see the screenshot below where some tests are failing.

![f1](https://user-images.githubusercontent.com/35504149/152643807-5cb7caa9-25ee-4668-ae9c-0c11071a0481.png)

![f2](https://user-images.githubusercontent.com/35504149/152643812-bbc30c72-5483-480a-8a44-1d8f3e590575.png)

Reply:
### Kindly note that all test are now passing
![5](https://user-images.githubusercontent.com/58692590/152647198-5c889a26-d2e2-49de-a9d4-1b3264acaadd.png)